source/mongodb-handshake/handshake.md

```diff
@@ -442,6 +446,12 @@ be appended to their respective fields, and be delimited by a `|` character. For
 }
 ```
 
+Some client libraries provide APIs that accept a pre-initialized MongoClient as an argument. In these circumstances, it
+is possible for multiple library objects to be associated with the same MongoClient, which could result in the same
+metadata being appended multiple times. Drivers MUST ensure that any duplicate `DriverInfoOptions` objects provided to a
```

Comment:
If we are using no-op language, it should be applied to the `appendMetadata`, so I recommend rephrasing the `Drivers MUST ensure that any duplicate ...` sentence to read: `Drivers MUST ensure that any repeated calls to appendMetadata with equivalent DriverInfoOptions are a no-op` (and, optionally, "resulting in no additional metadata being appended")